### PR TITLE
chore: updated readme and added docker compose for mongo

### PR DESCRIPTION
**Pull request by eacaps**
Adding some QOL stuff for devs getting started on the starter. Totally open to discussion/addition/removal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,23 +6,36 @@
 
 ## Getting Started
 
-1. Install Node module dependencies via: `npm install`
-1. Use the default configuration in `./config/env/default.js` or override with your own configuration that matches the `NODE_ENV` environment variable by copying the `./config/env/development.template.js` file and renaming it to match the value of `$NODE_ENV`
+### Prereqs
```
Comment (eacaps) [inline README.md:9]: I made some changes to the Getting Started section including installing mongo from docker(I had some crazy @swontonsoup clustered config and wanted to get back to the simplest install). All the other changes in this file are from prettier formatting

File: package.json [hunk 2]
```diff
@@ -13,6 +13,13 @@
     "yarn": ">=1.5.1"
   },
   "scripts": {
+    "init": "npm run init:nomongo && npm run init:db",
```
Comment (eacaps) [inline package.json]: I was considering flipping this and having the default `npm run init` assume you already have mongo. Thoughts?

Comment (jrassa) [reply package.json]: I would flip it.  Not all downstream projects are running mongo via docker.  Would then add some text to README.md describing the init:db scripts.

Comment (eacaps) [reply package.json]: flipped and added additional documentation

**Review by jrassa — APPROVED** [review]

**Pull request merged**